core/utils.py

```diff
@@ -94,6 +94,17 @@ class ExplorationConversionError(Exception):
     pass
 
 
+class ThrowGetError:
+    """Error class to throw error while invoking the get function."""
+
+    def __init__(self, message: str):
+        self.err_message = message
```

Comment:
Optional: err_message --> error_message (try to avoid abbreviations if not necessary)

Reply:
Yeah makes sense, changed it to `error_message`